src/publish.proj

```diff
@@ -36,6 +31,12 @@
     <OutDir>$(PackageDownloadDirectory)</OutDir>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PublishPattern Condition="'$(PublishPattern)' == '' and '$(RepublishSignedPackages)' == 'true'">$(FinalPublishPattern)</PublishPattern>
```

Comment:
It took me a bit to piece together what the paths mean here. A few things I *think* would help:

 * A comment in the `PropertyGroup` above saying what `Final` means vs. the `bin` paths.
 * Include "final" in the `RepublishSignedPackages` property name to tie them together. (`RepublishSignedFinalPackages` seems ok to me.)

However, it makes sense now.

Reply:
Just updated to address your feedback

Reply:
Thanks, LGTM.